cvat/schema.yml

```diff
@@ -1531,6 +1531,64 @@ paths:
       responses:
         '204':
           description: The invitation has been deleted
+  /api/invitations/{key}/accept:
+    post:
+      operationId: invitations_create_accept
+      parameters:
+      - in: path
+        name: key
+        schema:
+          type: string
+        description: A unique value identifying this invitation.
+        required: true
+      tags:
+      - invitations
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InvitationWriteRequest'
```

Comment:
Why does this accept a request body? You shouldn't need to supply anything to accept an invitation (except the identifier of the invitation, which is already passed via the URL path).

Reply:
As we discussed on HLD meeting, we should accept invitation and update user data in one request. Doing this operation in two or three requests can lead to additional problems wich we will have to handle. (invitation expired, deleted, user nickname is already user etc.)